mangaki/mangaki/models.py

```diff
@@ -185,19 +188,31 @@ def retrieve_tags(self, anidb):
 
         return {"deleted_tags": deleted_tags, "added_tags": added_tags, "updated_tags": updated_tags, "kept_tags": kept_tags}
 
-    def update_tags(self, deleted_tags, added_tags, updated_tags):
-        for title, weight in added_tags.items():
-            current_tag = Tag.objects.get_or_create(title=title)[0]
-            TaggedWork(tag=current_tag, work=self, weight=weight).save()
+    def update_tags(self, deleted_tags={}, added_tags={}, updated_tags={}):
```

Comment:
## Guess game:

```python
def add_new_tags(tags=[]):
    tags.append('blurgh')
    tags.append('fire')
    return tags

tags_1 = add_new_tags()
tags_2 = add_new_tags()
tags_3 = add_new_tags()

print(tags_3[4])
```

Can you guess the output of this chunk of code?

[Answer & Explanation.](http://python-guide-pt-br.readthedocs.io/en/latest/writing/gotchas/)